pkgs/data/icons/windows10-icons/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchurl }:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "windows10-icons";
+  version = "1.0";
+
+  src = fetchurl {
+    url = "https://github.com/B00merang-Artwork/Windows-10/archive/refs/tags/${finalAttrs.version}.tar.gz";
+    hash = "sha256-ZGuLyOBNOANIicqs7zhGy2ZVmV8YBdh7P/e8Cykkmq4=";
+  };
```

Comment:
Please use `fetchFromGitHub` instead :)

Reply:
I've no idea how that slipped me! :) Thank you, I'll make the change.